src/backend/db/videos.js

```diff
@@ -7,235 +7,296 @@ import { v4 as uuid } from "uuid";
 
 export const videos = [
   {
-    _id: uuid(),
-    title: "god of war",
+    _id: "-IJuKT1mHO8",
+    title: "far cry 6",
     image:
       "https://ik.imagekit.io/t007rushi/VideoCardImages/pc/farcry6_KgK7f2lbZE.png?ik-sdk-version=javascript-1.4.3&updatedAt=1649669798331",
```

Comment:
Maybe you could make a util function to generate this link as you have to keep repeating stuff here.